docs/mp/introduction.adoc

```diff
@@ -144,6 +144,18 @@ Helidon MP is an Eclipse MicroProfile {version-lib-microprofile-api} runtime tha
 
 !===
 
+== Migration
+
+In case you need to upgrade the version of Helidon, follow the `Migration Guides`.
```

Comment:
Use the following migration guides to upgrade your current Helidon version:

Reply:
Follow this guide to migrate your application from Helidon 2.x to 3.x:

Reply:
fixed